src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -2721,8 +2721,8 @@ namespace ZoneTempPredictorCorrector {
 
             for (Loop = 1; Loop <= state.dataGlobal->NumOfZones; ++Loop) {
                 FirstSurfFlag = true;
-                if (Zone(Loop).HTSurfaceFirst > 0) {
-                    for (SurfNum = Zone(Loop).HTSurfaceFirst; SurfNum <= Zone(Loop).HTSurfaceLast; ++SurfNum) {
+                if (state.dataHeatBal->Zone(Loop).HTSurfaceFirst > 0) {
```

Comment:
@xuanluo113 Is this line still necessary?

Reply:
it's okay to remove.